editor/plugins/script_text_editor.cpp

```diff
@@ -2425,6 +2426,9 @@ void ScriptTextEditor::register_editor() {
 	ED_SHORTCUT("script_text_editor/contextual_help", TTR("Contextual Help"), KeyModifierMask::ALT | Key::F1);
 	ED_SHORTCUT_OVERRIDE("script_text_editor/contextual_help", "macos", KeyModifierMask::ALT | KeyModifierMask::SHIFT | Key::SPACE);
 
+	ED_SHORTCUT("script_text_editor/lookup_symbol", TTR("Lookup Symbol"), Key::F12);
+	ED_SHORTCUT_OVERRIDE("script_text_editor/lookup_symbol", "macos", KeyModifierMask::CMD_OR_CTRL | Key::M);
```

Comment:
It would be good to do a review of what other popular IDEs use as hotkeys for this.

I'm always wary of picking arbitrary hotkeys for new shortcuts. For example here F12 will directly conflict with my drop-down terminal (yakuake). It's a me problem and I'll have to accept that I need to change this key binding every time I reset my Godot editor settings, which isn't too big a deal. But still we should have some confidence that those hotkeys are the ones users expect.

Reply:
Sure. AFAIK, Visual Studio use F12 for both mac and windows, Jetbrain Rider use Ctrl + B, which is conflict with godot's bookmark shortcut. VS is capable of both UE and Unity, so do Rider. So that might be resonable I think.